runtime/doc/lsp.txt

```diff
@@ -543,8 +543,7 @@ start_client({config})                                *vim.lsp.start_client()*
                     {root_dir}         (required, string) Directory where the
                                        LSP server will base its rootUri on
                                        initialization.
-                    {cmd}              (required, string or list treated like
-                                       |jobstart()|) Base command that
+                    {cmd}              (required, list) Base command that
```

Comment:
why remove the `treated like jobstart` ?

Reply:
I thought the `treated like jobstart` referred to the support of both `string` and `list`. The `jobstart()` function documentation explains that if it is a string it will use the shell for evaluation. 
That part no longer applies, so I felt it is confusing to still refer to `jobstart` which now has different semantics.

Reply:
but it also explains how the list is treated, eg. collapsed etc...

Reply:
Right, I pushed a fixup to add it back.